packages/desktop/src/renderer/app/constants/desktop-messages.constants.ts

```diff
@@ -292,5 +292,10 @@ export const DesktopMessages: {
     logPayload: messageParams,
     toastType: 'success',
     showToast: true
+  }),
+  CALLBACK_ERROR: (messageParams) => ({
+    message: `Error while invoking callback ${messageParams.error}!`,
+    log: true,
+    showToast: false
```

Comment:
You can remove this error message implementation as it will already be logged by the server lib. 
This is only for desktop app specific messages, or when we want to display a toast.